lib/Parser/Parse.cpp

```diff
@@ -6561,7 +6561,7 @@ bool Parser::ParseFncNames(ParseNodePtr pnodeFnc, ParseNodePtr pnodeFncParent, u
 
 void Parser::ValidateFormals()
 {
-    ParseFncFormals<false>(nullptr, nullptr, fFncNoFlgs);
+    ParseFncFormals<false>(this->m_currentNodeFunc, nullptr, fFncNoFlgs);
```

Comment:
Can you use this->GetCurrentFunctionNode() here instead? We should forbid accessing m_currentNodeFunc directly...

Reply:
done.